x/marker/keeper/querier.go

```diff
@@ -165,11 +165,11 @@ func accountForDenomOrAddress(ctx sdk.Context, keeper Keeper, lookup string) (ty
 	if addr, addrErr = sdk.AccAddressFromBech32(lookup); addrErr != nil {
 		account, err = keeper.GetMarkerByDenom(ctx, lookup)
 		// if we error on get by denom, ensure we also capture the fail to parse as an address
```

Comment:
this comment doesn't apply anymore

Reply:
Sorry, totally didn't notice that comment.